nncf/quantization/algorithms/smooth_quant/algorithm.py

```diff
@@ -359,7 +359,7 @@ def _process_weight_statistics(self, node: NNCFNode, weights: TTensor, port_id:
         """
         channel_axis = 0
         if len(weights.shape) > 1:
-            channel_axis = self._backend_entity.get_weight_channel_axis(node, port_id)
+            channel_axis = self._backend_entity.get_weight_channel_axis(node)
```

Comment:
We may drop the `port_id` method's attribute then.

Reply:
Thank you, done